<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmTempControl</class>
 <widget class="QMainWindow" name="FrmTempControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Temperature Control</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QGridLayout">
    <item row="3" column="0">
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="textLabel5_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Temp.</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="m_lcdSourceTemp">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="numDigits">
         <number>8</number>
        </property>
        <property name="mode">
         <enum>QLCDNumber::Dec</enum>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>3.000000000000000</double>
        </property>
        <property name="intValue" stdset="0">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="textLabel9_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>K</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="textLabel17">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Heater Mode</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_cmbHeaterMode"/>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="textLabel18_2_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>P</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_edP">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="textLabel18_2_2_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>I</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_edI">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="textLabel18_2_2_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>D</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_edD">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="textLabel16">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Source CH</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_cmbSourceChannel"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="textLabel5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Heater Power</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="m_lcdHeater">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="numDigits">
         <number>8</number>
        </property>
        <property name="mode">
         <enum>QLCDNumber::Dec</enum>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>3.000000000000000</double>
        </property>
        <property name="intValue" stdset="0">
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="textLabel18">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Target Temp</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_edTargetTemp"/>
      </item>
      <item>
       <widget class="QLabel" name="textLabel9_2_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>K</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="textLabel18_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Manual Heater</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_edManHeater"/>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="textLabel22">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Power Range</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_cmbPowerRange"/>
      </item>
     </layout>
    </item>
    <item row="8" column="0">
     <widget class="QGroupBox" name="groupBox1">
      <property name="title">
       <string>Channel Setup</string>
      </property>
      <layout class="QGridLayout">
       <property name="margin">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout">
         <item>
          <widget class="QLabel" name="textLabel19">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Setup Channel</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_cmbSetupChannel"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="textLabel20">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Excitation</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_cmbExcitation">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QVBoxLayout">
         <item>
          <widget class="QLabel" name="textLabel21">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Thermometer</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_cmbThermometer"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QGroupBox" name="groupBox2">
      <property name="title">
       <string>External DC Source</string>
      </property>
      <layout class="QGridLayout">
       <item row="1" column="1">
        <widget class="QComboBox" name="m_cmbExtDCSrcCh"/>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QComboBox" name="m_cmbExtDCSrc">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="textLabel1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Channel</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>m_cmbHeaterMode</tabstop>
  <tabstop>m_cmbSourceChannel</tabstop>
  <tabstop>m_edTargetTemp</tabstop>
  <tabstop>m_edManHeater</tabstop>
  <tabstop>m_cmbPowerRange</tabstop>
  <tabstop>m_edP</tabstop>
  <tabstop>m_edI</tabstop>
  <tabstop>m_edD</tabstop>
  <tabstop>m_cmbSetupChannel</tabstop>
  <tabstop>m_cmbExcitation</tabstop>
  <tabstop>m_cmbThermometer</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
